### PR TITLE
fix index.md & add png files at "tensorflow/tensorflow/g3doc/images" 

### PR DESCRIPTION
**Pull request by hephaex**
When I review TF tutorial using Github website at index.md on "tensorflow/tensorflow/g3doc/tutorials/mnist/beginners/index.md"
There are no image on index.md and others 

I think there are no png files. If I possible to upload png files on 

for example "tensorflow/tensorflow/g3doc/tutorials/mnist/beginners/index.md"

Line 18: 
 --> there is no MNIST.png at "../../../images/MNIST.png"

Line 67: 
 --> there is no MNIST-Matrix.png at "../../../images/MNIST-Matrix.png"

Line 81: 
 --> there is no mnist-train-xs.png at "../../../images/mnist-train-xs.png"

Line103: 
 --> there is no mnist-train-ys.png at "../../../images/mnist-train-ys.png"

Line124: 
 --> there is no softmax-weights.png at "../../../images/softmax-weights.png"

Line181: 
 --> there is no softmax-regression-scalargraph.png at "../../../images/softmax-regression-scalargraph.png"

Line187: 
 --> there is no softmax-regression-scalarequation.png at "../../../images/softmax-regression-scalarequation.png"

Line195: 
 --> there is no softmax-regression-vectorequation.png at "../../../images/softmax-regression-vectorequation.png"

### CODE REVIEW

**Comment by martinwicke** [comment]
We kept the images out of the repo, they are instead served from tensorflow.org. We keep the markdown in github so you can fix things, but it's not meant to be viewed on github.
